Components/Button/button.css

```diff
@@ -32,6 +32,46 @@
   padding: 0.6rem 1.2rem;
 }
 
+.ghost-btn {
+  height: min-content;
+  padding: 5px;
+  border-bottom: 2px solid var(--neon-green-color);
+}
+.ghost-btn:hover {
+  transition: all 0.2s ease-in-out;
+  border-bottom: 2px solid transparent;
+  background-color: var(--neon-color-light);
+}
+
+.subtle-btn {
+  height: min-content;
+  border: 1px solid var(--grey);
+  padding: 2px 5px;
+  border-radius: 3px;
+  color: var(--dark-grey);
+}
+.subtle-btn:hover {
+  border: 1px solid var(--violet-color);
+}
+
+.link-btn {
+  height: min-content;
+  padding: 0px;
```

Comment:
A recommendation to use just 0 instead of 0px/0rem. It's a better practice!

Reply:
okay! will change it

Reply:
okay will do!!